app/components/concept-page/question-card.hbs

```diff
@@ -4,14 +4,27 @@
       {{markdown-to-html @question.queryMarkdown}}
     </div>
 
-    <div class="flex flex-col -mx-3 sm:-mx-4 gap-px" {{did-insert this.handleDidInsertOptionsList}}>
+    {{! template-lint-disable no-invalid-interactive }}
+    <div
+      class="question-card-option-container flex flex-col -mx-3 sm:-mx-4 gap-px"
+      {{did-insert this.handleDidInsertOptionsList}}
+      {{! @glint-expect-error on-key modifier types aren't right? }}
+      {{on-key "k" this.handleMoveUp}}
+      {{! @glint-expect-error on-key modifier types aren't right? }}
+      {{on-key "j" this.handleMoveDown}}
+      {{! @glint-expect-error on-key modifier types aren't right? }}
+      {{on-key "ArrowUp" this.handleMoveUp}}
+      {{! @glint-expect-error on-key modifier types aren't right? }}
+      {{on-key "ArrowDown" this.handleMoveDown}}
+      {{on "keydown" this.handleKeydown}}
```

Comment:
@libmartinito looks like we're still implementing keydown here? can't this be done using `on-key` too? Note that `on-key` doesn't need to be used as a modifier, it can be used as a regular helper too. 

So you can do something like: 

```
{{#each this.optionShortcuts as |shortcut|}}
  {{on-key shortcut.key (fn this.handleOptionShortcut(shortcut.optionIndex)}}
{{/each}}
```

(Naming could be better there, just wanted to illustrate how you can add dynamic shortcuts)
{{/each}}